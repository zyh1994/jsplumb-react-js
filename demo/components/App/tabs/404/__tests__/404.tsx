import React from 'react';
import renderer from 'react-test-renderer';
import E404 from '../404';

test('Error 404 page', () => {
  const component = renderer.create(
    <E404 />
  );
  const e404 = component.toJSON();
  expect(e404).toMatchSnapshot();
});
