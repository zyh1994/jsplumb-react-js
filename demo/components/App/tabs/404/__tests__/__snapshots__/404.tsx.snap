// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error 404 page 1`] = `
<div
  className="jspr-E404"
>
  <h4>
    404
  </h4>
</div>
`;
